core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -55,8 +55,7 @@
     <li>
       <ul *ngIf="!windowIsNarrow"
           class="nav oppia-navbar-tabs">
-        <li *ngIf="CLASSROOM_PROMOS_ARE_ENABLED"
-            [ngClass]="{'open' : activeMenuName === 'classroomMenu'}"
+        <li [ngClass]="{'open' : activeMenuName === 'classroomMenu'}"
```

Comment:
Can we please change the name from `classroomMenu` to `learnMenu` maybe? or some other better name as it is not just classroom menu and also contains community library link. It will also be confusing to developer that why is this menu visible if CLASSROOM_PROMOS_ARE_ENABLED is false? Ditto at other places where we are talking about the dropdown menu as a whole rather than just the classroom.

Reply:
Done